.travis.yml

```diff
@@ -7,17 +7,13 @@ addons:
   firefox: latest
   google-chrome: latest
 
-before_script:
-  - gulp
-  - bundlesize -f dist/sweetalert2.all.min.js -s 15kB
-
 script:
-  # unit tests against dist/sweetalert2.js
-  - xvfb-run testem ci --reporter dot
+  # xvfb-run is needed for headless testing with Chrome and Firefox
+  - xvfb-run npm test
 
-  # unit tests against dist/sweetalert2.all.min.js
+  # repeat unit tests against dist/sweetalert2.all.min.js
   - sed -i 's/sweetalert2.js/sweetalert2.all.min.js/g' testem.json
-  - xvfb-run testem ci --reporter dot
+  - xvfb-run npm run check:tests
```

Comment:
please revert to `npm test` for a reason I mentioned below.

Reply:
Which reason are you referring to? The only one I see above (about reverting the before_script section) seems unrelated

Reply:
sorry, this one: https://github.com/sweetalert2/sweetalert2/pull/841/files#r163172838

Reply:
> build is build, test is test

Ah, this line never said `npm test`.. But yes if I need to change the name of `check:test` I will update this line accordingly.